packages/paste-icons/svg/ChatIcon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Mind removing `Icon` from the file names of the SVG files please? To be consistent with the folder naming scheme